src/integration/browser.ts

```diff
@@ -58,20 +58,16 @@ export const hasRecentlyLoggedIn = callOnce(() => {
   return lastLoginAt + BROWSER_LAST_SESSION_EXPIRATION > Date.now()
 })
 
-export const hasRecentlyDownloadModalShown = callOnce(() => {
+export const hasDownloadModalShown = callOnce(() => {
   const lastLoginAt = Number(localStorage.getItem(BROWSER_LAST_DOWNLOAD_MODAL_SHOWN_KEY))
-  if (Number.isNaN(lastLoginAt)) {
-    return false
-  }
-
-  return lastLoginAt + BROWSER_LAST_SESSION_EXPIRATION > Date.now()
+  return !(Number.isNaN(lastLoginAt) || !lastLoginAt)
```

Comment:
I would only use:

```suggestion
  return !!lastLoginAt
```

and remove any other check